  <launch>
  <param name="robot_description" command="cat $(find au_dynamixel_example)/conf/au_dmx_robot.urdf" />
  
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="rob_st_pub" output="screen" >
	<remap from="joint_states" to="/arm_controller/joint_state"/>
	<remap from="robot_description" to="/robot_description" />
	
    </node>
    
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
        <param name="use_gui" value="true" />
        <remap from="joint_states" to="/arm_controller/joint_state"/>
    </node>
    
    <node pkg="rviz" name="rvizz" type="rviz"></node>
  </launch>
